common/src/model_deployment/deploy.py

```diff
@@ -0,0 +1,161 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""Run Model deployment module."""
+import os
+import argparse
+from pathlib import Path
+import json
+from azure.ai.ml import MLClient
+from azure.ai.ml.entities import (
+    ManagedOnlineEndpoint,
+    ManagedOnlineDeployment,
+    OnlineRequestSettings,
+)
+from azure.identity import ManagedIdentityCredential
+from azureml.core import Run
+
+
+def parse_args():
+    """Return arguments."""
+    parser = argparse.ArgumentParser()
+
+    # add arguments
+    parser.add_argument(
+        "--registration_details",
+        type=str,
+        help="Text file that contains the ID of registered model to be deployed",
+    )
+    parser.add_argument(
+        "--endpoint_name",
+        type=str,
+        help="Name of the endpoint",
+    )
+    parser.add_argument(
+        "--deployment_name", type=str, help="Name of the the deployment"
+    )
+    parser.add_argument(
+        "--instance_type",
```

Comment:
instance_type validation?

Reply:
Thanks for the suggestion. Changes it to enum type